src/gateway/router/routes/deployContractRoute.ts

```diff
@@ -137,13 +137,7 @@ export async function deployContractRoute(ctx: Router.RouterContext) {
       await gatewayDb('contracts_src').insert(contracts_src_insert).onConflict('src_tx_id').ignore();
     }
 
-    sleep(2000)
-      .then(() => {
-        sendNotificationToCache(ctx, contractTx.id, initState);
-      })
-      .catch((e) => {
-        logger.error(`No sleep 'till Brooklyn.`, e);
-      });
+    sendNotification(ctx, contractTx.id, {initState, tags: contractTags});
```

Comment:
I'm pretty sure that `contractTags` are decoded - they come from the `prepareTags` function, which handles the decoding